contribs/gmf/examples/layertree.html

```diff
@@ -133,10 +133,31 @@
       .treenode a.expand-node.fa[aria-expanded="true"]::before {
           content: "\f078";
       }
+
+      /** Disclaimer */
+      gmf-disclaimer {
+        position: absolute;
+        bottom: 0.5rem;
+        left: 0.5rem;
+        width: 30rem;
+      }
+      gmf-disclaimer .alert {
+        padding: 0.5rem 1rem;
+        margin: 0 0 0.5rem 0;
+      }
+      gmf-disclaimer .alert-dismissable .close,
+      gmf-disclaimer .alert-dismissible .close {
+        right: -5px;
```

Comment:
Effectively that bootstrap use px is an issue :-/ ...
Using relative lengths let us to easly zoom in the page
http://www.w3schools.com/cssref/css_units.asp